server/service/src/sync/translations/stocktake_line.rs

```diff
@@ -10,12 +10,23 @@ use repository::{
     SyncBufferRow,
 };
 use serde::{Deserialize, Serialize};
-use util::sync_serde::{date_option_to_isostring, empty_str_as_option_string, zero_date_as_option};
+use util::sync_serde::{
+    date_option_to_isostring, empty_str_as_option_string, object_fields_as_option,
+    zero_date_as_option,
+};
 
 use super::{
     utils::clear_invalid_location_id, PullTranslateResult, PushTranslateResult, SyncTranslation,
 };
 
+#[allow(non_snake_case)]
+#[derive(Deserialize, Serialize)]
+pub struct TransStocktakeLinRowOmsFields {
```

Comment:
As an OGer `Trans` is `transact` and this struct name confused me for a minute when I first saw it in `server/service/src/sync/test/test_data/stocktake_line.rs`. I presume you were going for `translator`?

I'd rather it just called Legacy consistent with the other legacy translator structs so we know exactly what it is. Also there was a missing `e`
```suggestion
pub struct LegacyStocktakeLineRowOmsFields {
```

Reply:
Opps that was a failed copy-pasta hahaha